src/modules/announcements/components/AnnouncementItem.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import styles from "./announcementItem.module.scss";
+
+export const AnnouncementItem = ({ post }) => {
```

Comment:
I don't understand what is `AnnoucementItem` component. It receives a single `post` as a prop but maps it. How?